### PR TITLE
Artem/vampir aux fun

### PR DESCRIPTION
**Pull request by agureev**
Create `bool` and `next-range` functions and parameters.

### CODE REVIEW

File: src/vampir/spec.lisp [hunk 1]
```diff
@@ -202,3 +202,71 @@
...
+								:lhs (make-constant :const 2)
+								:rhs (make-wire :var :a1))))
+		    (make-tuples :wires (list
+					 (make-wire :var :a0)
+					 (make-application :func :range
+						           :arguments (make-wire :var :a1))))))))
+(defun next-range (range x)
+          (make-application :func :next-range
+                                :arguments
+                                (list range x)))
```
Comment (mariari) [inline src/vampir/spec.lisp:272]: This does not belong in the spec file, should go to a different file. Likely `vampir.lisp` and we move all the `print-object` methods to `print.lisp` in the same folder

Comment (rokopt) [reply src/vampir/spec.lisp]: I've tried the code movement you suggested, as well as rebasing this branch up to main.  Would you mind re-reviewing to see if we can get this in now?  Thanks!

File: test/poly.lisp [hunk 2]
```diff
@@ -1,9 +1,82 @@
...
+
+(define-test vampir-test-pair-bool
+  :parent geb-poly
+  (of-type geb.vampir.spec:alias pair-bool-circuit))
+
+(def fst-bool-circuit
+  (stlc-to-vampir
+   nil bool
+   (geb.lambda.spec:fst bool bool pair-bool-stlc)
+   :tc_fst_bool))
```
Comment (mariari) [inline test/poly.lisp:82]: These seem to relate more towards vampir or lambda or pipeline testing

File: test/lambda-conversion.lisp [hunk 3]
```diff
@@ -26,18 +26,18 @@
...
-      (conversion:stlc-ctx-to-mu (list geb-bool:bool))
+      (geb.lambda.trans:stlc-ctx-to-mu (list geb-bool:bool))
       (geb:prod geb-bool:bool geb:so1)
       "fold singleton bool context"))
 
 (define-test fold-multi-object-context :parent compile-checked-term
   (is obj-equalp
-      (conversion:stlc-ctx-to-mu  (list geb-bool:bool geb:so0 geb:so1))
+      (geb.lambda.trans:stlc-ctx-to-mu  (list geb-bool:bool geb:so0 geb:so1))
       (geb:prod geb-bool:bool (geb:prod geb:so0 (geb:prod geb:so1 geb:so1)))
```
Comment (mariari) [inline test/lambda-conversion.lisp:42]: we should probably still be using aliasing, I can fix this up when I merge it

File: src/vampir/vampir.lisp [hunk 4]
```diff
@@ -132,45 +132,45 @@ of ()'s for any non normal form"
...
+                           (spc:make-wire :var :a0)
+                           (spc:make-application :func :range
                                              :arguments
-                                             (list (make-wire :var :a1)))))))))
+                                             (list (spc:make-wire :var :a1)))))))))
 
 (defun next-range (range x)
-  (make-application :func :next_range
+  (spc:make-application :func :next_range
                     :arguments (list range x)))
```
Comment (mariari) [inline src/vampir/vampir.lisp:50]: vampir should have access to the spec so this shouldn't be needed

**Review by mariari — CHANGES_REQUESTED** [review]
Going to some changes myself to fix up the comments

**Comment by mariari** [comment]
After the force push everything should be dandy

**Pull request merged**